chain/jsonrpc/res/debugz.html

```diff
@@ -0,0 +1,7 @@
+<html>
+
+<body>
+    <h1><a href="/debugz/last_blocks">Last blocks</a></h1>
+</body>
+
+</html>
```

Comment:
trailing newline missing

Reply:
done - but my editor keeps removing it..